src/main/java/org/jmhsrobotics/frc2024/subsystems/LED/LEDSubsystem.java

```diff
@@ -0,0 +1,67 @@
+package org.jmhsrobotics.frc2024.subsystems.LED;
+
+import edu.wpi.first.wpilibj.AddressableLED;
+import edu.wpi.first.wpilibj.AddressableLEDBuffer;
+import edu.wpi.first.wpilibj.RobotBase;
+import edu.wpi.first.wpilibj.simulation.AddressableLEDSim;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class LEDSubsystem extends SubsystemBase {
+	private AddressableLED LED;
+	private AddressableLEDBuffer ledBuffer;
+
+	private int rainBowPixelHue = 0;
+	public LEDSubsystem() {
+		if (RobotBase.isSimulation()) {
+			initSim();
+		}
+
+		// TODO: find the real port
+		this.LED = new AddressableLED(9);
+
+		// TODO: get the real length
+		this.ledBuffer = new AddressableLEDBuffer(60);
```

Comment:
Move Values for Ports and LED length into a constants file/class

Reply:
I created a new issue to create a global constant file and do the clean up